### ISSUE TITLE
Compile multiple .asms into a single file?

### ISSUE THREAD

**Issue by starjake** [pret/pokecrystal#488]
I'm not looking to compile a ROM; I'm just trying to compile information into a single database. Under pokecrystal/data/pokemon/base_stats/, there is a .asm for every pokemon, and I'd like to put the contents of all those into one file. Is there a way to do that other than opening each one individually and copypasting the data?

**Comment by Rangi42** [comment]
Look up Cygwin's `cat` utility.

`cat data/pokemon/base_stats/*.asm > all_base_stats.txt`

**Comment by yenatch** [comment]
or in order: `grep INCLUDE data/pokemon/base_stats.asm | sed 's/.*"\(.*\)".*/\1/g') | xargs cat`

**Comment by starjake** [comment]
I read the cat utility manual, but neither of the provided cat or grep commands provided worked.

When I entered cat data/pokemon/base_stats/*.asm > all_base_stats.txt, I got:
cat: 'data/pokemon/base_stats/*.asm': No such file or directory

When I entered grep INCLUDE data/pokemon/base_stats.asm | sed 's/.*"\(.*\)".*/\1/g') | xargs cat > all_base_stats.txt, I got:
-bash: syntax error near unexpected token `)'

**Comment by i0brendan0** [comment]
You forgot the `*` in `data/pokemon/base_stats/*.asm`. You also forgot some `*`'s in `grep INCLUDE data/pokemon/base_stats.asm | sed 's/.*"\(.*\)".*/\1/g') | xargs cat > all_base_stats.txt`

**Comment by yenatch** [comment]
oops
`grep INCLUDE data/pokemon/base_stats.asm | sed 's/.*"\(.*\)".*/\1/g') | xargs cat > all_base_stats.txt`

**Comment by starjake** [comment]
I'm still getting -bash: syntax error near unexpected token `)' when I try the grep command? I'm copypasting it directly from the above comment, so I can't be making a typo

**Comment by starjake** [comment]
I must have been copying the wrong text before. I'm no longer getting a syntax error, but I am getting a "No such file or directory" error.

![image](https://user-images.githubusercontent.com/36341624/36067556-82a03236-0e7c-11e8-81a9-7f2033256be7.png)

**Comment by i0brendan0** [comment]
because you need to be inside the pokecrystal directory.

**Comment by starjake** [comment]
So it turns out I missed a couple steps when I downloaded Cygwin. I uninstalled it and am now re-installing it, and it is stuck in the place shown in the screenshot below.

![image](https://user-images.githubusercontent.com/36341624/36081486-bf8a7176-0f5c-11e8-888c-b6b3bf5e9b1c.png)

Reminder that the goal here is to access the information in a manner where I don't have to load each .asm file individually (not necessarily to make Cygwin work).
